pydra/engine/workers.py

```diff
@@ -356,13 +356,7 @@ class DaskWorker(Worker):
     def __init__(self, **kwargs):
         """Initialize Worker."""
         super().__init__()
-        try:
-            from dask.distributed import Client
-        except ImportError:
-            logger.critical("Please instiall Dask distributed.")
-            raise
```

Comment:
this import is in init so it gets caught at the point of initialization of the worker, not when the worker is executed, which is an async function.

Reply:
this move seems to be done for styling rather than code. you can mark something to be ignored by styling if that is a problem.

Reply:
ok, will work on flask settings, not on the code